docs/source/dsa.rst

```diff
@@ -42,7 +41,7 @@ remain inaccessible to the public as no original data are shared.
 
 
 Withdrawing records and period of agreement
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+...........................................
 
```

Comment:
```suggestion
```
I'll stop here, but please go ahead and remove all of these empty lines whenever possible. I left some that I can't modify without sending you a PR (e.g., line 9 above)